crates/swc_ecma_transforms_base/src/helpers/_ts_decorate.js

```diff
@@ -1,4 +1,4 @@
-var __decorate = (this && this.__decorate) || function (decorators, target, key, desc) {
+var _ts_decorate = (this && this._ts_decorate) || function (decorators, target, key, desc) {
```

Comment:
I found that I had to change export name in order to make the ts inline helpers work. 
From my personal opinion, I don't particularly like this change. It makes our code look different from the ts helpers.

Reply:
I agree witg you, but the problem was the SyntaxContext